src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtils.java

```diff
@@ -0,0 +1,224 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.api.services.bigquery.model.TableRow;
+import com.google.gcp_variant_transforms.common.Constants;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLineCount;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Utility methods for write values with defined value type into Big Query rows.
+ */
+public class VariantToBqUtils {
+  public static String addReferenceBase(VariantContext variantContext) {
```

Comment:
To be honest, I am a bit confused why it should be "reference_bases" instead of "reference_base" since it only returns a single string. I will add one "s" in the naming to match to BQ field